.github/workflows/publish.yml

```diff
@@ -21,12 +21,12 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install setuptools wheel twine
+        pip install build twine
     - name: Build and publish
       if: startsWith(github.ref, 'refs/tags')
       env:
         TWINE_USERNAME: __token__
         TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
       run: |
-        python setup.py sdist bdist_wheel
+        python -m build
         twine upload dist/*
```

Comment:
Maybe we could simplify this code by copying it [from Scrapy](https://github.com/scrapy/scrapy/blob/master/.github/workflows/publish.yml).

Reply:
Cool, will pick it up in the next commit